### ISSUE TITLE
Sprint 4 – Scrum Meeting (23-12-2016)

### ISSUE THREAD

**Issue by lkc442012** [mewind/RSDSupermarketPOS#25]
Hello guys! Since yesterday Chun Xiong have uploaded the last part of the code in the system, that is the generation of exception and summary report, our system can be considered done now!

**Comment by huylin25** [comment]
Wow, finally we have done it!

**Comment by mewind** [comment]
Thank you everyone that take part in this assignment!

**Comment by lkc442012** [comment]
Although the system is done, but we still need to do some finalization to make sure that we fulfill all the user requirements.

**Comment by juzplayplay** [comment]
You are right.  We need to have a review on our system and the output of each iteration to ensure that we have finish every part of the task.

**Comment by huylin25** [comment]
I agree with you. We cannot miss out any part in the user requirement since it will cause a serious problem in the future if we did not notice about it.

**Comment by lkc442012** [comment]
Apart from that, I will also write out an activity log that records all the member's activities, daily discussion main point, feedback and also the project development schedule. This can helps us to have a look about what we have done in the whole development process.

**Comment by mewind** [comment]
Thanks Kar Cheng. Through this activity log, I think we can know what is the output that have been produced in each day. This may helps us in the improvement of schedule planning for the next coming project.

**Comment by huylin25** [comment]
Yea. This documentation can actually provides a view for the developers to better manage the project.

**Comment by juzplayplay** [comment]
Kar Cheng, do you need our help in doing the final report?

**Comment by lkc442012** [comment]
Hmm, I think it would not be a problem for me to finish it. Don't worry, I will in charge of it.

**Comment by juzplayplay** [comment]
Okay, thank you very much!

**Comment by huylin25** [comment]
Welcome! I think we can dismiss the meeting now since there is not much task to be assign to you guys.

**Comment by mewind** [comment]
Okay, thanks Kar Cheng!
